tools/mo/openvino/tools/mo/ops/split.py

```diff
@@ -213,7 +213,7 @@ def infer(node):
         out_shape[axis] = input_shape[axis] // num_splits
 
         input_value = node.in_port(0).data.get_value()
-        output_value = np.split(input_value.copy(), axis=axis, indices_or_sections=num_splits) \
+        output_value = np.split(input_value.copy(), axis=axis, indices_or_sections=mo_array(num_splits, dtype=np.int)) \
```

Comment:
How `num_split` can be of `float` type if it is just retrieved from attribute?

Reply:
I added here just in case, to be sure. Do you want me to revert it?

Reply:
I added here just in case, to be sure. Because if for some reason num_splits will be float (e.g. if someone manually created in transformation node with such a float attribute) then numpy will fails and we user will have huge backtrace with unfriendly error message.

But if you insist to revert it back I will do that.

Reply:
I think it is better to remove it. Honestly, now it looks ugly. If `num_split` is floating-point, it means someone did mistake in transformation or extractor by initializing `num_split` to `float`

Reply:
done